src/brevitas/graph/equalize.py

```diff
@@ -173,8 +173,8 @@ def _combine_weights_bias(
         return weight.data
 
     bias = bias.data
-    weight = deepcopy(weight).view(weight.shape[0], -1)
-    bias = deepcopy(bias).view(-1, 1)
+    weight = weight.reshape(weight.shape[0], -1)
```

Comment:
can we do a view instead of a reshape?

Reply:
No for some models/configurations it fails with view